code/qcommon/ojk_saved_game_class_archivers.h

```diff
@@ -19,10 +19,7 @@ template<>
 class SavedGameClassArchiver<cplane_t>
 {
 public:
-	static constexpr bool is_implemented()
-	{
-		return true;
-	}
+	enum { is_implemented = true };
```

Comment:
Valid but looks like a "hack".
Isn't static constant field would be more readable?
(i.e., "static const bool is_implemented = true;")